### DISCUSSION TITLE
Understanding the found architecture

### DISCUSSION THREAD

**Discussion by SimonsThijs** [keras-team/autokeras#1600] [Q&A]
After running autokeras on my dataset I obtain the following hyperparameters:

    "values": {
      "image_block_1/normalize": true,
      "image_block_1/augment": false,
      "image_block_1/block_type": "vanilla",
      "classification_head_1/spatial_reduction_1/reduction_type": "flatten",
      "classification_head_1/dropout": 0,
      "optimizer": "adam",
      "learning_rate": 0.0001,
      "image_block_1/conv_block_1/kernel_size": 7,
      "image_block_1/conv_block_1/separable": false,
      "image_block_1/conv_block_1/max_pooling": true,
      "image_block_1/conv_block_1/dropout": 0.25,
      "image_block_1/conv_block_1/num_blocks": 1,
      "image_block_1/conv_block_1/num_layers": 2,
      "image_block_1/conv_block_1/filters_0_0": 32,
      "image_block_1/conv_block_1/filters_0_1": 64,
      "image_block_1/conv_block_1/filters_1_0": 256,
      "image_block_1/conv_block_1/filters_1_1": 128,
      "image_block_1/conv_block_1/filters_2_0": 512,
      "image_block_1/conv_block_1/filters_2_1": 32
    }

Now i want to recreate this neural net to do some extra validation. However, i struggle to understand the exact architecture these hyperparameters represent. Especially the different amount of filters. Could someone explain to me how i should interpret these hyperparameters in order for me to recreate the architecture?

**Comment by ghost** [comment]
I think this is a super important question when it comes to recreation of the architecture, can anybody help?
